### ISSUE TITLE
 Focus state colours on GOV.UK Review application legacy pages

### ISSUE THREAD

**Issue by nickcolley** [alphagov/govuk-frontend#1400]
This issue is from a May 2019 external accessibility audit report.

**WCAG Reference**: 1.4.11 Non-text Contrast (Level AA)
**Issue ID**: DAC_Issue16
**URL**: Frontend pages

## Summary
There were some elements present that had a colour used for styling focus indication that did not pass the expected ratio of 3:1

## Screen Shot
![](https://user-images.githubusercontent.com/2445413/58710113-811c2200-83b3-11e9-972f-f52716fc8740.png)

The colour of the focus indicator used on some pages failed to meet the expected ratio of 3:1 to pass WCAG 2.1 success criteria 1.4.11 Non-text Contrast AA and may prove difficult for some users to perceive.

## Current Code Ref(s)
**Outline**: #ffbf47
**Background**: #ffffff
**Ratio**: 1.64:1

## Screen Shot 2
![](https://user-images.githubusercontent.com/2445413/58710121-85e0d600-83b3-11e9-802c-bc4d503fec5c.png)

In some instances, the colour had been used as an entire background to the text which passed the colour contrast limitations at 12.8:1.

## Screen Shot 3
![](https://user-images.githubusercontent.com/2445413/58710129-8bd6b700-83b3-11e9-9849-c7c6e045321f.png)

The new colour that has been used on some areas of the Frontend pages does now pass the expected ratio, as the background colour changes as opposed to the outline and meets 15.6:1.

## Solution

Ensure that all outline colour properties available for focus indication, meet the ratio of 3:1. The colour of the ‘outline’ property must meet 3:1 against the background (white).

**Comment by nickcolley** [comment]
In the review application we intentionally keep GOV.UK Elements page examples around to be able to test the compatibility with GOV.UK Frontend.

This issue was raised against that old code which we expected to fail, and confirms that the new styles do not fail in the same way.

With this in mind I am going to close this as resolved as the GOV.UK Frontend styles do not have issues with focus state contrast.
